client/src/pages/FlightData/tabs/Quick.js

```diff
@@ -96,37 +96,40 @@ const Quick = () => {
 					</Row>
 					<Row>
 						<Box label="Altitude"
-							 content={AaltitudeIsGlobal ? AaltitudeGlobal.toFixed(2) + " ft MSL" : Aaltitude.toFixed(2) + " ft AGL"}
-							 onClick={() => {setAaltitudeIsGlobal(!AaltitudeIsGlobal)}}
-							style={{ cursor: "pointer" }} />
+							content={AaltitudeIsGlobal ? AaltitudeGlobal.toFixed(2) + " ft MSL" : Aaltitude.toFixed(2) + " ft AGL"}
+							onClick={() => {setAaltitudeIsGlobal(!AaltitudeIsGlobal)}}
+							style={{ cursor: "pointer" }}
+							title="The plane's altitude. MSL refers to above mean sea level. AGL is the height from the ground where the plane is." />
 						<Box label="Battery (6S)" content={Abattery.toFixed(2) + "V"} />
 					</Row>
 				</Row>
 				<Row style={{ gap: "1rem" }}>
 					<Row>
 						<Box label="Ground Speed"
-							 content={((AspeedIsInKnots ? 0.868976 : 1) * AgroundSpeed).toFixed(2) + (AspeedIsInKnots ? " knots" : " mph")}
-							 onClick={() => {setAspeedIsInKnots(!AspeedIsInKnots)}}
-							 style={{ cursor: "pointer" }} />
+							content={((AspeedIsInKnots ? 0.868976 : 1) * AgroundSpeed).toFixed(2) + (AspeedIsInKnots ? " knots" : " mph")}
+							onClick={() => {setAspeedIsInKnots(!AspeedIsInKnots)}}
+							style={{ cursor: "pointer" }}
+							title="Speed over only the surface of the earth, not taking height into account." />
 						<Box label="Airspeed"
-							 content={((AspeedIsInKnots ? 0.868976 : 1) * Aairspeed).toFixed(2) + (AspeedIsInKnots ? " knots" : " mph")}
-							 onClick={() => {setAspeedIsInKnots(!AspeedIsInKnots)}}
-							 style={{ cursor: "pointer" }} />
+							content={((AspeedIsInKnots ? 0.868976 : 1) * Aairspeed).toFixed(2) + (AspeedIsInKnots ? " knots" : " mph")}
+							onClick={() => {setAspeedIsInKnots(!AspeedIsInKnots)}}
+							style={{ cursor: "pointer" }}
+							title="Speed of the plane overall, including change in altitude." />
```

Comment:
That's not what airspeed is - it's speed data from sensors on the plane (relative to wind speed)